tests/fast-integration/skr-test/commerce-mock/index.js

```diff
@@ -0,0 +1,103 @@
+const {
+  checkFunctionResponse,
+  checkInClusterEventDelivery,
+  sendLegacyEventAndCheckResponse,
+  sendCloudEventStructuredModeAndCheckResponse,
+  sendCloudEventBinaryModeAndCheckResponse,
+  deleteMockTestFixture,
+  ensureCommerceMockWithCompassTestFixture,
+} = require('../../test/fixtures/commerce-mock');
+const {
+  AuditLogCreds,
+  AuditLogClient,
+  checkAuditLogs,
+} = require('../../audit-log');
+const {debug} = require('../../utils');
+const {director} = require('../provision/provision-skr');
+
+const AWS_PLAN_ID = '361c511f-f939-4621-b228-d0fb79a1fe15';
+
+// const auditLogsThreshold = 4;
+const testTimeout = 1000 * 60 * 30; // 30m
+
+// prepares all the resources required for commerce mock to be executed;
+// runs the actual tests and checks the audit logs in case of AWS plan
+function commerceMockTest(options) {
```

Comment:
Suggestion: Maybe better to use explanatory function names instead of comments, because the name can be read everywhere it's used and comments can get noisy/redundant:  e.g.`commerceMockTests` here for the whole suite and see below other suggestion